client/packages/dashboard/src/widgets/StockWidget.tsx

```diff
@@ -19,31 +19,28 @@ const LOW_MOS_THRESHOLD = 3;
 
 export const StockWidget: React.FC = () => {
   const modalControl = useToggle(false);
-  const { error } = useNotification();
+  const { error: errorNotification } = useNotification();
   const t = useTranslation('dashboard');
   const formatNumber = useFormatNumber();
-  const { data: expiryData, isLoading: isExpiryLoading } =
-    useDashboard.statistics.stock();
-  const { data: itemCountsData, isLoading: isItemStatsLoading } =
-    useDashboard.statistics.item(LOW_MOS_THRESHOLD);
-  const [hasExpiryError, setHasExpiryError] = React.useState(false);
-  const [hasItemStatsError, setHasItemStatsError] = React.useState(false);
-
-  React.useEffect(() => {
-    if (!isExpiryLoading && expiryData === undefined) setHasExpiryError(true);
-    return () => setHasExpiryError(false);
-  }, [expiryData, isExpiryLoading]);
-
-  React.useEffect(() => {
-    if (!isItemStatsLoading && itemCountsData === undefined)
-      setHasItemStatsError(true);
-    return () => setHasItemStatsError(false);
-  }, [itemCountsData, isItemStatsLoading]);
+  const {
+    data: expiryData,
+    error: expiryError,
+    isLoading: isExpiryLoading,
+    isError: hasExpiryError,
+  } = useDashboard.statistics.stock();
+  const {
+    data: itemCountsData,
+    error: itemCountsError,
+    isLoading: isItemStatsLoading,
+    isError: hasItemStatsError,
+  } = useDashboard.statistics.item(LOW_MOS_THRESHOLD);
 
   const { mutateAsync: onCreate } = useRequest.document.insert();
   const onError = (e: unknown) => {
     const message = (e as Error).message ?? '';
-    const errorSnack = error(`Failed to create requisition! ${message}`);
+    const errorSnack = errorNotification(
+      `Failed to create requisition! ${message}`
```

Comment:
Maybe a little ToDo to localise ?

Reply:
oops. was just renaming `error` and didn't think about the actual message. will do.